feature-libs/commerce-quotes/styles/_commerce-quotes-list.scss

```diff
@@ -1,18 +1,45 @@
 %cx-commerce-quotes-list {
   @extend .cx-agnostic-table;
   table {
+    th {
+      text-transform: capitalize;
+      text-align: start;
+      &:nth-last-of-type(1) {
+        text-align: start;
+      }
+    }
     tr {
       &:hover {
         background-color: var(--cx-color-background);
+        cursor: pointer;
       }
     }
-
-    a {
-      color: var(--cx-color-text);
+    td {
+      text-align: start;
+      &:nth-last-of-type(1) {
+        text-align: start;
+      }
     }
 
-    .cx-commerce-quotes-list-quote-name > a {
-      text-decoration: underline;
+    tbody {
+      a {
+        color: var(--cx-color-text);
+        text-decoration: none;
+      }
+      .quote-draft {
+        color: #788082;
```

Comment:
Did you check if there are variables for these colours?

Reply:
I did a global search. I didn't find those colours. I'm not sure where else to check. Could you kindly recommend?